### PR TITLE
Add 2023 Scenarios

### PR DESCRIPTION
**Pull request by ianna**
- Add 2023dev and 2023sim scenarios dump

### CODE REVIEW

**Comment by ianna** [comment]
please test

File: Fireworks/Geometry/python/dumpSimGeometry_cfg.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import FWCore.ParameterSet.VarParsing as VarParsing
 from FWCore.Utilities.Enumerate import Enumerate
 
-varType = Enumerate ("Run1 Ideal2015 Ideal2015dev 2015 2015dev 2019 PhaseIPixel Phase1_R34F16 Phase2Tk  2023Muon MaPSA CRack PhaseIIFlatTracker SLHC DB SLHCDB")
+varType = Enumerate ("Run1 Ideal2015 Ideal2015dev 2015 2015dev 2019 PhaseIPixel Phase1_R34F16 Phase2Tk  2023dev 2023sim 2023Muon MaPSA CRack PhaseIIFlatTracker SLHC DB SLHCDB")
```
Comment (davidlange6) [inline Fireworks/Geometry/python/dumpSimGeometry_cfg.py]: seems that a bunch of these are obsolete and should be removed? @ianna

**Comment by ianna** [comment]
please test

**Comment by alja** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**